apps/desktop/src/lib/stores/user.ts

```diff
@@ -15,6 +15,19 @@ export type LoginToken = {
 	url: string;
 };
 
+/** Holds the logged in user's token in memory */
+export class TokenMemoryService {
+	#token = writable<string | undefined>(undefined);
+
+	get token() {
+		return this.#token;
+	}
+
+	setToken(data: string | undefined) {
+		this.#token.set(data);
+	}
+}
+
```

Comment:
I've had to move the token into its own little store in the Desktop app because it needs to be used by both the HttpService and the UserService, while the UserService also depends on the HttpService

```mermaid
flowchart TD
  UserService --> HttpService
  HttpService --> TokenMemoryService
  UserService --> TokenMemoryService
```